calibration/FLORTM/CGINS-FLORTM-03186__20191025.csv

```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
```

Comment:
Where is this cal sheets on doris so I can double check?

Reply:
doris gliders:data/386/000012/8304-61003-386-00012_Configuration_Record.xlsx